### DISCUSSION TITLE
在添加第十个时间线节点时无效

### DISCUSSION THREAD

**Discussion by Yizhixixi** [Class-Widgets/Class-Widgets#186] [Q&A问答] [answered]
### **设置 时间线编辑 节点 在添加第十个时间线节点时点击+无效**

**Comment by pizeroLOL** [comment, marked as answer]
这个是因为配置文件的限制，`时间线编辑-时间线` 的保存格式是 `/[a,f][0-8][0-9]+/`，第一位 a 为课程，f 为课间，第二位 [0-8] 则是 `时间线编辑-节点` 的编号，第三位及以后则是 `时间线编辑-时间线` 的编号。也正是这个问题导致了 #184 。

**Reply by RinLit-233-shiroko** [reply]
dd，以后会在软件界面做提示的
